pages/Android-Interfaces.md

```diff
@@ -0,0 +1,9 @@
+### Android Interfaces (Cargo)
```

Comment:
Articles cannot start with a heading, so it would be ideal if you could perhaps start with one or two introduction sentences before diving into the two variants.